scripts/node.sh

```diff
@@ -101,17 +108,25 @@ fi
 # find my public ip address
 myip
 
-# public beacon node multiaddress
-BC_MA=/ip4/54.183.5.66/tcp/9999/ipfs/QmdQVypu6NSm7m8bNZj5EJCnjPhXR8QyRmDnDBidxGaHWi
+# public boot node multiaddress
+BN_MA=/ip4/100.26.90.187/tcp/9876/p2p/QmZJJx6AdaoEkGLrYG4JeLCKeCKDjnFz2wfHNHxAqFSGA9,/ip4/54.213.43.194/tcp/9876/p2p/QmQayinFSgMMw5cSpDUiD9pQ2WeP6WNmGxpZ6ou3mdVFJX
 
 if [ "$OS" == "Linux" ]; then
 # Run Harmony Node
-   nohup ./harmony -bc_addr $BC_MA -ip $PUB_IP -port $NODE_PORT > harmony-${PUB_IP}.log 2>&1 &
+   LD_LIBRARY_PATH=$(pwd) nohup ./harmony -bootnodes $BN_MA -ip $PUB_IP -port $NODE_PORT -is_beacon > harmony-${PUB_IP}.log 2>&1 &
 else
-   ./harmony -bc_addr $BC_MA -ip $PUB_IP -port $NODE_PORT > harmony-${PUB_IP}.log 2>&1 &
+   DYLD_FALLBACK_LIBRARY_PATH=$(pwd) ./harmony -bootnodes $BN_MA -ip $PUB_IP -port $NODE_PORT -is_beacon > harmony-${PUB_IP}.log 2>&1 &
 fi
 
+echo "############### Running Harmony Process ###############"
+ps -ef | grep harmony | grep -v grep
+echo
+echo
+
 echo Please run the following command to inspect the log
 echo "tail -f harmony-${PUB_IP}.log"
 
+echo
+echo You may use \"sudo kill -9 harmony\" to terminate running harmony node program.
```

Comment:
Now that we are about to persist DB contents, let's not suggest `kill -9` but simply `kill`.

`kill -9` will prevent whatever cleanup routine (typically bound to `SIGTERM` handler) from running, and the next time the node starts up, the DB opening procedure may find it necessary to replay the intent logs.